src/AccessibilityInsights.Core/Types/TypeBase.cs

```diff
@@ -115,5 +115,10 @@ public bool Exists(int id)
         {
             return this.Dic.ContainsKey(id);
         }
+
+        /// <summary>
+        /// the values of the types contained in the inheriting class
+        /// </summary>
+        public IEnumerable<int> Values => Dic.Keys;
```

Comment:
Should this be called Types?

```suggestion
        public IEnumerable<int> Types => Dic.Keys;
```

Reply:
I don't think so. I consider this to be the values of the types.